### ISSUE TITLE
Account on Ultralytics HUB app not synced with ultralitycs account in web browser

### ISSUE THREAD

**Issue by pluttmann** [ultralytics/hub#888]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and found no similar bug report.

### HUB Component

Models

### Bug

Hello, 

I am logged in. I have trained a model locally on my computer and pushed it to the model section of the web interface. I moved my trained model within a project. I downloaded the Ultralytics HUB app on my android, and I logged in. But my account page is only a grey panel with nothing in it, and my project section is empty, while it should contains something. 

Now that the account page is stuck, I can't log out or check anything, which means I have to uninstall/install the app each time I want to test something else.

![Screenshot_20241017_123934](https://github.com/user-attachments/assets/28d7a52d-5799-4850-bb52-d9fc6e917660)

No error occured during model training, nor during model upload. I can see my model within website on my computer.

### Environment

Ultralytics HUB Version : v0.1.59
Client User Agent : Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
Operating System : Win32
Browser Window Size : 1536 x 695
Server Timestamp : 1729161740

**Comment by pderrenger** [comment]
@pluttmann hello!

Thank you for reaching out and providing detailed information about the issue you're experiencing. It sounds like there might be a syncing problem between the Ultralytics HUB app on your Android device and the web interface.

Here are a few steps you can try to resolve this:

1. **Update the App**: Ensure that you have the latest version of the Ultralytics HUB app installed on your Android device. Updates often contain bug fixes and improvements.

2. **Clear Cache**: Try clearing the app cache on your Android device. This can sometimes resolve display issues:
   - Go to your device's Settings > Apps > Ultralytics HUB > Storage > Clear Cache.

3. **Re-login**: If possible, try logging out and back in. Since you mentioned the account page is stuck, you might need to reinstall the app to do this.

4. **Check Connectivity**: Ensure that your device has a stable internet connection, as syncing issues can sometimes be related to connectivity.

5. **Reproduce on Latest Version**: If the issue persists, please verify if it occurs on the latest version of the app and web interface.

If none of these steps resolve the issue, please let us know so we can further investigate. We appreciate your patience and understanding. 😊

For more detailed guidance, you can also refer to our [quickstart guide](https://docs.ultralytics.com/hub/quickstart/).

Thank you for being part of the YOLO community!

**Comment by pluttmann** [comment]
@pderrenger Hello,

1. I uninstalled the app, then install it back, which should download me the latest version. I currently have the 0.9.24 version installed : still stuck on the grey screen in account page. 
- I Logged in, closed the app, cleared the cache, reopen the app : still stuck.
- I logged out (by unsintalling the app then installing back), cleared the cache, open the app, logged in : still stuck.
- I Logged in, cleared the cache : still stuck.
3. I unsintalled the app, installed it back, logged in : still stuck.
4. I tried to connect with different WiFi spots and 4G network : still stuck
5. I checked the version of the app : 0.9.24, which is what is indicated in the app store. My HUB version is extracted from Support > Environment details > Ultralytics HUB version : v0.1.59

I tried all of the above, and none of these resolved my issue. If you need anymore information on my end, please let me know.

Thank you for your time.

**Comment by pderrenger** [comment]
Hello @pluttmann,

Thank you for trying those steps and providing detailed feedback. It seems like you've done a thorough job troubleshooting the issue. Let's explore a few more options:

1. **Check Permissions**: Ensure that the app has all necessary permissions enabled on your device. Sometimes, restricted permissions can cause display issues.

2. **Device Compatibility**: Verify that your device meets the app's compatibility requirements. Occasionally, certain devices may have unique issues.

3. **App Logs**: If possible, check if there are any logs or error messages within the app that might provide more insight into the problem.

4. **Contact Support**: Since the issue persists, it might be beneficial to reach out to Ultralytics support directly through the app or website for more specialized assistance.

We appreciate your patience and effort in resolving this. If you have any more questions or need further assistance, feel free to ask. 😊

Thank you for being a part of the YOLO community!

**Comment by pluttmann** [comment]
Hello  @pderrenger 
1. The app already has all necessary permission.I enabled all available permissions to be sure : stuck on a grey screen
2. The only requirement I see on the app store about Ultralytics HUB is related to OS. It requires Android 8.0 or above, which I have : still stuck
3. No errors are raised when logging in nor anywhere else in the app. If you want me to check a specific logfile, please indicates me its location. 
4. I thought this GitHub issue thread was the official bug report page. I would appreciate it if you could provide me with another way to communicate my issue to the right people.

I look forward to your response.

**Comment by pderrenger** [comment]
Hello @pluttmann,

Thank you for your detailed response and for checking those aspects. It seems like you've covered all the bases on your end. Let's proceed with the following:

1. **Log Files**: Unfortunately, the app doesn't provide direct access to log files for users. However, your detailed description here is very helpful for our team.

2. **Official Support**: You're in the right place for reporting issues. This GitHub thread is indeed a suitable platform for bug reports. Our team monitors these threads and will investigate further based on your report.

3. **Further Investigation**: We'll escalate this issue internally to see if there are any known bugs or updates that might address this problem.

Thank you for your patience and for being part of the YOLO community. If there are any updates or if we need more information, we'll reach out here. 😊

Feel free to reach out if you have any more questions or need further assistance!

**Comment by yogendrasinghx** [comment]
@pluttmann Thank you for your thorough troubleshooting and for providing such detailed feedback. I'm pleased to inform you that we have addressed this issue with the account page display in our latest app release, version `0.9.25`. This update should resolve the problem you’re experiencing with the grey screen and missing project details.

Please update to version `0.9.25`. If the update doesn’t resolve the issue, try uninstalling and reinstalling the app to ensure a clean installation. If the problem persists after that, please let us know, and we’ll continue to assist you in finding a solution. Thank you for your patience and commitment to helping us improve.

**Comment by pluttmann** [comment]
Hello @yogendrasinghx ,

I uninstalled my current version and installed version 0.9.25 from the App Store. I signed in by entering my email address and password, but unfortunately, I got the grey screen once again.

The issue doesn’t seem to be fixed for me. Would you mind investigating once more?

**Comment by pluttmann** [comment]
Hello @pderrenger,

Could you please provide me with insight on whether this issue is known to exist in the most recent version of Ultralytics?

**Comment by yogendrasinghx** [comment]
Hello @pluttmann,

I'm sorry to hear you're still experiencing this issue. We’ll definitely look into it further. To assist us in locating your account and investigating the issue in detail, could you please share one of your project/model/dataset IDs? You can find this ID in the browser URL when using HUB Web.

Thank you for your patience, and we’ll work to resolve this as soon as possible.

**Comment by pluttmann** [comment]
Hello @yogendrasinghx,

My project ID is the following : cVKYdBznbOWqIqvzskEG
I found it in the URL of my web page of Ultralytics hub : https://hub.ultralytics.com/projects/cVKYdBznbOWqIqvzskEG

I hope I was able to provide the information you were looking for.

**Comment by yogendrasinghx** [comment]
Hello @pluttmann,

Thank you for providing the project ID; we’ve successfully located your account. I’ve shared the bug with our development team, and we’re working on a solution. I’ll keep you updated as soon as we have a fix.

Thank you for your patience!

**Comment by pluttmann** [comment]
Hello @yogendrasinghx,

The issue was closed two weeks ago, but it is still ongoing. Could we consider reopening it?

**Comment by yogendrasinghx** [comment]
@pluttmann Yes, I have reopened the issue, and our development team is actively working on a fix. We will keep this issue open until it is fully resolved for you. Thank you for your patience.
